### PR TITLE
[TS] LPS-157886 Every fieldset is showing twice in global in structures

### PR DESCRIPTION
**Pull request by IstvanD**
Dear Team,

Could you please review this pull request?

Thank you and best regards,
István

### CODE REVIEW

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by IstvanD** [comment]
Dear Team,

The test failures are not related to the solution.

Best regards,
István

**Review by miltonmc — CHANGES_REQUESTED** [review]
Thank you so much for this fix!

I suggest you to do the following tweak:

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/FormBuilder.js [hunk 1]
```diff
@@ -75,9 +75,13 @@ export function FormBuilder() {
...
-			const groupFieldSetsPromise = getItem(
-				`/o/data-engine/v2.0/data-definitions/by-content-type/${contentType}`
-			);
+			let groupFieldSetsPromise = [];
+
+			if (groupId !== themeDisplay.getCompanyGroupId()) {
+				groupFieldSetsPromise = getItem(
+					`/o/data-engine/v2.0/data-definitions/by-content-type/${contentType}`
+				);
+			}
```
[the remark this answers is not in the document]
Comment (IstvanD) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/FormBuilder.js]: hi, @miltonmc thank you for the suggestions!
I have made the changes and force-pushed them to this pr.

**Review by miltonmc — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**